spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/amqp/RabbitProperties.java

```diff
@@ -699,6 +699,21 @@ public void setConsumersPerQueue(Integer consumersPerQueue) {
 		 */
 		private Long replyTimeout;
 
+		/**
+		 * The name of the default exchange to use for send operations.
```

Comment:
Configuration properties description do not start with"The", "A", etc. Please look at the appendix to harmonize them.